client/packages/common/src/types/schema.ts

```diff
@@ -800,38 +800,33 @@ export type DocumentRegistryConnector = {
 };
 
 export type DocumentRegistryFilterInput = {
-  context?: InputMaybe<EqualFilterDocumentRegistryContextInput>;
+  documentContext?: InputMaybe<EqualFilterStringInput>;
   documentType?: InputMaybe<EqualFilterStringInput>;
   id?: InputMaybe<EqualFilterStringInput>;
   parentId?: InputMaybe<EqualFilterStringInput>;
+  type?: InputMaybe<EqualFilterDocumentRegistryTypeInput>;
 };
 
 export type DocumentRegistryNode = {
   __typename: 'DocumentRegistryNode';
   children: Array<DocumentRegistryNode>;
-  context: DocumentRegistryNodeContext;
+  documentContext: Scalars['String'];
   documentType: Scalars['String'];
   formSchemaId: Scalars['String'];
   id: Scalars['String'];
   jsonSchema: Scalars['JSON'];
   name?: Maybe<Scalars['String']>;
   parentId?: Maybe<Scalars['String']>;
+  type: DocumentRegistryTypeNode;
```

Comment:
This was the old `context` which is now the type of the document registry entry, e.g. Patient | Encounter | ProgramEnrolment